README.adoc

```diff
@@ -1,34 +1,125 @@
-////
-This is the base template for Hazelcast integration module guides.
-
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
-
-:github-address: [github-repo-address-of-your-guide]
+:github-address: https://github.com/hazelcast-guides/getting-started-hazelcast-springboot
 :templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+:hazelcast: Hazelcast IMDG
+:framework: Spring Boot
 
-= Title
+= Getting Started with Hazelcast using Spring Boot
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+This guide will help you to start using Hazelcast by building a Spring Boot application.
+
+include::{templates-url}/what-is-hz.adoc[]
```

Comment:
Do we really need it? I think someone who wants to learn how to set up Hazelcast with Spring Boot, already knows what Hazelcast is. To me this looks a little weird.

![image](https://user-images.githubusercontent.com/2834997/78642665-622fcb80-78b3-11ea-9cf6-ef5d2896de07.png)

Reply:
I agree. We should direct people to hazelcast documentation or maybe to that video. 
https://www.youtube.com/watch?time_continue=1&v=c8qEkyfgeek

Reply:
Removed 👍